pages/index.tsx

```diff
@@ -316,125 +64,13 @@ export const Home: React.FC<HomeProps> = ({
-                <HeroCard
-                  key={title}
-                  variant={variant as HeroCardVariantType}
-                  title={title}
-                  description={description}
-                  cta={
-                    <Button
-                      variant={BUTTON_VARIANT.SECONDARY}
-                      href={callToAction.href}
-                    >
-                      {callToAction.title}
-                    </Button>
-                  }
-                >
-                  {filesCollection.items.map(({ title: fileTitle, url }) => {
-                    return (
-                      <HeroCardChild key={fileTitle}>
-                        <a href={url}>Download {fileTitle.toLowerCase()}</a>
-                        &nbsp;&nbsp;
-                        <IconFileDownload />
-                      </HeroCardChild>
-                    )
-                  })}
-                  {code && (
-                    <HeroCardChild>
-                      <HeroCardChild>
-                        <StyledDollar>$</StyledDollar>
-                        {code.sourceCode}
-                      </HeroCardChild>
-                    </HeroCardChild>
-                  )}
-                </HeroCard>
-              )
-            }
-          )}
-        </StyledHeroCards>
-      </Section>
-      <Section sectionIndex="2" title={section2Heading}>
-        <p>{section2SubHeading}</p>
-        {section2Buttons.map(({ href, title }) => {
-          return (
-            <Button key={title} href={href}>
-              {title}
-            </Button>
-          )
-        })}
-      </Section>
-      <Section sectionIndex="3" title={section3Heading}>
-        <p>{section3SubHeading}</p>
-        <StyledCards>
-          {section3CardsCollection?.items.map(
-            ({ title, titleColor, buttonHref, buttonTitle, description }) => {
-              return (
-                <Card
-                  key={title}
-                  title={title}
-                  titleColor={titleColor}
-                  href={buttonHref}
-                  anchorText={buttonTitle}
-                >
-                  {description}
-                </Card>
-              )
-            }
-          )}
-        </StyledCards>
-      </Section>
-      <Section sectionIndex="4" title={section4Heading}>
-        <p>{section4SubHeading}</p>
-        {section4Buttons.map(({ href, title }) => {
-          return (
-            <Button key={title} href={href}>
-              {title}
-            </Button>
-          )
-        })}
-      </Section>
-      <Section sectionIndex="5" title={section5Heading}>
-        <p>{section5SubHeading}</p>
-        <StyledButtonGroup>
-          {section5Buttons.map(({ href, title }) => {
-            return (
-              <Button key={title} href={href}>
-                {title}
-              </Button>
-            )
-          })}
-        </StyledButtonGroup>
+
+      <Section title="301 Moved Permanently">
```

Comment:
Can we change this to be more succinct:

![Screenshot 2024-04-29 at 08 55 24](https://github.com/Royal-Navy/design-system-docs/assets/48086589/db8f3e9e-f28e-4613-bd4c-db4b986b14b6)